### PR TITLE
Intial commit on the sentence creator

### PR DESCRIPTION
**Pull request by lbk003**

### CODE REVIEW

File: sentencecreator/.gitignore [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+#   install all needed dependencies.
+#Pipfile.lock
+
+# poetry
+#   Similar to Pipfile.lock, it is generally recommended to include poetry.lock in version control.
+#   This is especially recommended for binary packages to ensure reproducibility, and is more
+#   commonly ignored for libraries.
+#   https://python-poetry.org/docs/basic-usage/#commit-your-poetrylock-file-to-version-control
+#poetry.lock
+
```
Comment (MySmilingTurban) [inline sentencecreator/.gitignore:103]: can remove these comments

File: sentencecreator/sentencecreator.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
+import openai
+import pandas as pd
+import os
+import config
+import click
+import csv
+
+# Set up your OpenAI API credentials
+
```
Comment (MySmilingTurban) [inline sentencecreator/sentencecreator.py:9]: remove empty line

File: sentencecreator/sentencecreator.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        messages=messages,
+        temperature=config.TEMPERATURE,
+    )
+    return response.choices[0].message["content"]
+
+
+def create_prompt(wordlist):
+    prompt = f"Here is a list of words:{wordlist}"
+    sentence_count = config.SENTENCES_PER_WORD
+    prompt += f"For each of these words, create {sentence_count} sentences using the word for a grade 3 student."
```
Comment (MySmilingTurban) [inline sentencecreator/sentencecreator.py:26]: Line too long, should have max 100 characters

File: sentencecreator/sentencecreator.py [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+    prompt += f"Draft the response as a comma delimited list that can be saved as a csv file. Column 0 is the word, Column 1 through {sentence_count} are the {sentence_count} sentences generated for that word."
```
Comment (MySmilingTurban) [inline sentencecreator/sentencecreator.py:27]: Line too long, should have max 100 characters

File: sentencecreator/sentencecreator.py [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+def main(filename):
+    wordlist = read_wordlist(filename)
+    prompt = create_prompt(wordlist)
+    response: str = get_completion(prompt)
+    out_filename = "out_" + filename
+    save_wordlist(out_filename, response)
+
+
+if __name__ == "__main__":
+    main()
```
Comment (MySmilingTurban) [inline sentencecreator/sentencecreator.py:76]: nit: add new line

**Review by MySmilingTurban — CHANGES_REQUESTED** [review]
added suggestions for few minor changes
can add pylint and docstrings, if needed

File: sentencecreator/sentencecreator.py (hunk 2 above)
Comment (lbk003) [reply sentencecreator/sentencecreator.py:9]: Done

File: sentencecreator/sentencecreator.py (hunk 4 above)
Comment (lbk003) [reply sentencecreator/sentencecreator.py:27]: done

File: sentencecreator/sentencecreator.py (hunk 5 above)
Comment (lbk003) [reply sentencecreator/sentencecreator.py:76]: done

**Comment by lbk003** [comment]
> added suggestions for few minor changes can add pylint and docstrings, if needed

Will definitely do that when we bump up the quality from poc

**Review by MySmilingTurban — APPROVED** [review]
Looks good to me!

**Pull request merged**